### PR TITLE
docs: add samples from logging/cloud-client

### PR DESCRIPTION
**Pull request by busunkim96**
Samples from https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/logging/cloud-client

### CODE REVIEW

**Review by daniel-sanche — APPROVED** [review]
LGTM. I fixed the failing unit/system tests here: https://github.com/googleapis/python-logging/pull/70. Still working on the last docs check that's failing, then hopefully we can merge these together

What's the long term plan for samples? Will we remove them out of `python-docs-samples` after this is merged? Do we need to update all the docs to point to the new samples home too?

**Comment by busunkim96** [comment]
Thanks @Daniel-Sanche!

> What's the long term plan for samples? Will we remove them out of python-docs-samples after this is merged? Do we need to update all the docs to point to the new samples home too?

Yes and yes. Once this is merged I'll (1) create a CL to point the docs here and (2) remove the samples from python-docs-samples.

You can ignore `docs-presubmit` if that's the only thing that's failing. It is for the cloud docs folks but is not a required check quite yet.

**Comment by Daniel-Sanche** [comment]
Ok, yeah it looks like it might be a bit of work to get the docs test to pass, so probably better as a separate PR. My test PR should be ready to review now then!

**Review by tmatsuo — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # The @googleapis/yoshi-python is the default owner for changes in this repo
-*               @googleapis/yoshi-python
+*               @simonz130 @daniel-sanche @googleapis/yoshi-python
```
Comment (simonz130) [inline .github/CODEOWNERS:8]: Please change it to @googleapis/api-logging

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
 
 # The python-samples-reviewers team is the default owner for samples changes
-/samples/  @googleapis/python-samples-owners
\ No newline at end of file
+/samples/  @simonz130 @daniel-sanche @googleapis/python-samples-owners
```
Comment (simonz130) [inline .github/CODEOWNERS:11]: Please change it to @googleapis/api-logging

**Pull request merged**